diff --git a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
index 441442f..f862742 100644
--- a/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
+++ b/node_modules/eslint-plugin-react-hooks/cjs/eslint-plugin-react-hooks.development.js
@@ -9,11 +9,9 @@
  */
 
 'use strict';
-
 if (process.env.NODE_ENV !== "production") {
   (function() {
-'use strict';
-
+    'use strict';
 function _unsupportedIterableToArray(o, minLen) {
   if (!o) return;
   if (typeof o === "string") return _arrayLikeToArray(o, minLen);
@@ -905,6 +903,16 @@ var ExhaustiveDeps = {
         var _callee = callee,
             name = _callee.name;
 
+        if (name === 'useRefMiddleware') {
+            // useRefMiddleware() return value is stable.
+            if (id.type === 'ArrayPattern' && id.elements.length === 2 && isArray(resolved.identifiers)) {
+              if (id.elements[0] === resolved.identifiers[0]) {
+                return true;
+              }
+            }
+          return false;
+        }
+
         if (name === 'useRef' && id.type === 'Identifier') {
           // useRef() return value is stable.
           return true;
